packages/paypal-fastlane-integration/src/PayPalFastlaneShippingAddressForm.tsx

```diff
@@ -64,11 +64,11 @@ const PayPalFastlaneShippingAddressForm = (props: PayPalFastlaneStaticAddressPro
     };
 
     const braintreeFastlaneOptions = {
-        // braintreefastlane: {
-        //     onPayPalFastlaneAddressChange: (showPayPalFastlaneAddressSelector: any) => {
-        //         paypalFastlaneShippingComponent.current.showAddressSelector = showPayPalFastlaneAddressSelector;
-        //     },
-        // },
+        braintreefastlane: {
+            onPayPalFastlaneAddressChange: (showPayPalFastlaneAddressSelector: any) => {
```

Comment:
Can you add a type `PayPalFastlaneAddressComponentRef['showAddressSelector']` instead of any, please.